collections/misc/protectedspecies.php

```diff
@@ -153,23 +154,23 @@ function submitAddSpecies(f){
 	</div>
 	<div style="clear:both">
 		<section class="fieldset-like">
-			<h1><span>Global Protections</span></h1>
+			<h1><span><?= $LANG['GLOBAL_PROTECTIONS'] ?></span></h1>
 			<br/>
 			<?php
 			if($isEditor){
 				?>
 				<div class="editobj" style="display:none;width:400px;margin-bottom:20px">
 					<form name="addspeciesform" action='protectedspecies.php' method='post' >
 						<fieldset style='margin:5px'>
-							<legend><b>Add Taxon to List</b></legend>
+							<legend><b><?= $LANG['ADD_TAXON'] ?></b></legend>
 							<div style="margin:3px;">
-								Scientific Name:
+								<?= $LANG['SCIENTIFIC_NAME'] ?>:
```

Comment:
Should be in a `<label>` tag. This page has already gone through 508 compliance; it would be nice to avoid having to do it again.